socomo-maven/pom.xml

```diff
@@ -18,7 +18,7 @@
 
 	<properties>
 		<!-- keep required maven version low so that even old projects can use socomo -->
-		<maven.version>3.2.5</maven.version>
+		<maven.version>3.9.5</maven.version>
```

Comment:
Some projects that use Socomo still use old versions of Maven. So this should be kept as low as possible, to not break those projects. If the socomo-maven-plugin is build using never version, there's a risk that those old project will no longer be able to use it.